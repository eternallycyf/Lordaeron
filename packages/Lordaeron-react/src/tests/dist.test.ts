describe('test web-react pkg', () => {
  it('test pc dist cjs module correctly', () => {
    const lordaeron = require('../../web-react/cjs');
    expect(Object.keys(lordaeron)).toMatchSnapshot();
  });

  it('test pc dist esm module correctly', () => {
    const lordaeron = require('../../web-react/es');
    expect(Object.keys(lordaeron)).toMatchSnapshot();
  });

  it('test pc dist umd module correctly', () => {
    const lordaeron = require('../../web-react/umd');
    expect(Object.keys(lordaeron)).toMatchSnapshot();
  });
});

export { };
