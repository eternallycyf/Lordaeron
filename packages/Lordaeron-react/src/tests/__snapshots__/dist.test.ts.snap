// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test web-react pkg test pc dist cjs module correctly 1`] = `
Array [
  "Button",
]
`;

exports[`test web-react pkg test pc dist esm module correctly 1`] = `
Array [
  "Button",
]
`;

exports[`test web-react pkg test pc dist umd module correctly 1`] = `
Array [
  "Button",
]
`;
